sql/migrate/lex.go

```diff
@@ -174,10 +174,21 @@ func (l *lex) addPos(p int) {
 }
 
 func (l *lex) skipQuote(quote rune) error {
+	pos := l.pos
 	for {
 		switch r := l.next(); {
 		case r == eos:
-			return fmt.Errorf("unclosed quote %q", quote)
+			var (
+				s    = l.input[:pos]
+				col  = strings.LastIndex(s, `\n`)
+				line = 1 + strings.Count(s, `\n`)
+			)
+			if line == 1 {
+				col = pos
+			} else {
+				col = pos - col - 2
+			}
+			return fmt.Errorf("unclosed quote %q at %d:%d", quote, line, col)
```

Comment:
Maybe add an `error/f` method and report this format for all errors:

```go
file:%d:%d: <message>
```
`file` can be constant or omitted as we don't have this info.